### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by ghost** [lcn2/calc#109] [Q&A] [answered]
I installed it to `/usr`. I want to uninstall it to reinstall to `/usr/local`.

**Comment by lcn2** [comment, marked as answer]
Did you try `make uninstall`?

**Reply by ghost** [reply]
I don't know that `make uninstall` exists.

**Comment by lcn2** [comment]
A `make uninstall` using the same parameters as you used with `make install` should remove everything installed. 

By same parameters we mean if you made changes on the command line such as:

```sh
make install TOPDIR=/foo
```

Then you will need to use:

```sh
make uninstall TOPDIR=/foo
```

The same goes for if you modified Makefile variables, say via `Makefile.local`.
